apps/i18n/common/ru_ru.json

```diff
@@ -1683,7 +1683,7 @@
   "topRight": "вверху справа",
   "tos": "Условия предоставления услуг",
   "tosLong": "На этот проект поступала жалоба по поводу нарушения [Terms of Service]({url}) Code.org, и им нельзя делиться с другими пользователями.",
-  "totalNumLinesOfCodeWritten": "Общее количество за все время: {numLines, plural, one {1 line} other {# lines}} кода.",
+  "totalNumLinesOfCodeWritten": "Общее количество за все время: {numLines, plural, one {1 line} других {# lines}} кода.",
```

Comment:
@castro-jorge a translator broke formatting here

Reply:
oh, this is one of the translations I fixed last week. I might've missed updating this one in Crowdin